src/js/table.js

```diff
@@ -168,19 +170,29 @@ function reallyDisplayOrders(orders, beautiful) {
             cols.forEach( col_spec => {
                 let elem = null;
                 switch(col_spec.type) {
+                    // This seems to be only for when info is available already and no initial prep is needed.
+                    // Seems like the item description could use this also.
                     case 'plain':
                         elem = addCell(tr, order[col_spec.property_name]);
                         break;
+                    // Used for amounts.
                     case 'detail':
                         elem = addCell(tr, 'pending');
+                        elem.setAttribute('style', tableStyle + 'text-align:right');
```

Comment:
do this in css - we've got a stylesheet for it.
If you cannot address it (compose a selector)  conveniently from the css, then feel free to add a class (azad_detail_cell) and select the style on that.